packages/pods-core/src/builtin/__tests__/MarkdownPod.spec.ts

```diff
@@ -78,4 +78,129 @@ describe("MarkdownPublishPod", () => {
       }
     );
   });
+
+  test("schema with template", async () => {
+    await runEngineTestV5(
+      async ({ engine, wsRoot, vaults }) => {
+        const vault = vaults[0];
+        const vaultName = VaultUtils.getName(vault);
+        const fnames = ["bar.template.ch1", "bar.template.ch2"];
+        const resp = await Promise.all(
+          fnames.map(async (fname) => {
+            return new MarkdownPublishPod().execute({
+              engine,
+              vaults,
+              wsRoot,
+              config: {
+                fname,
+                vault: vaultName,
+                dest: "stdout",
+              },
+            });
+          })
+        );
+        expect(resp).toEqual(["ch1 template", "ch2 template"]);
+      },
+      {
+        expect,
+        preSetupHook: ENGINE_HOOKS.setupSchemaPreseet,
+      }
+    );
+  });
+
+  test("schema with namespace template", async () => {
```

Comment:
same comment as above. this tests for schema functionality but doesn't test something specific about Publish